src/main/java/edu/kis/powp/jobs2d/drivers/RecordingDriverDecorator.java

```diff
@@ -12,13 +12,14 @@ public RecordingDriverDecorator(Job2dDriver driver) {
     }
     @Override
     public void setPosition(int x, int y) {
-        job2dDriver.setPosition(x,y);
+        //Should it look like this?
+        //job2dDriver.setPosition(x,y);
```

Comment:
What are these comments for?

Reply:
If you want to resign from decorator. Change class name and remove job2dDriver field